### PR TITLE
Create calibration package

### PR DESCRIPTION
**Pull request by jdebacker**
This PR reorganizes scripts in the repo, creates new functions, and other adjustments with the goal of creating an OG-USA calibration python package.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- pytest-pep8
+- pytest-xdist
+- pycodestyle
+- pylint
+- coverage
+- requests
+- xlwt
+- statsmodels
+- linearmodels
+- rpy2
```
Comment (MaxGhenis) [inline environment.yml:33]: does R also need to be a dep?

**Review by MaxGhenis — COMMENTED** [review]
@prrathi said installing `rpy2` from conda wasn't sufficient to get the PSID R code running

**Comment by jdebacker** [comment]
@MaxGhenis It's not. 

I can look into how to deal with R packages in the environment, but the `psid1968to2017.RData` file is checked into the repo and that is what is created with the `psid_download.R` script.  So for the `ogusa-calibrate` package, there will be no need to run R (although I think we'll continue to want to have the `paid_download.R` script in the repo to update the RData file and to have code that takes one all the way from the raw PSID to the parameter calibrations based on it).

**Comment by jdebacker** [comment]
@MaxGhenis and @prrathi -- FYI, the `rpy2` is only used to read the R data frame into pandas in `psid_data_setup.py`.   To run the `psid_download.R` script one needs to have installed R and the `devtools` and `openxlsx` libraries.

**Pull request merged**